scripts/mve.py

```diff
@@ -62,45 +54,38 @@ def process(self, inputs, outputs):
             # mve makescene wants the output directory
             # to not exists before executing it (otherwise it
             # will prompt the user for confirmation)
-            if io.dir_exists(tree.smvs):
-                shutil.rmtree(tree.smvs)
+            if io.dir_exists(tree.mve):
+                shutil.rmtree(tree.mve)
 
             # run mve makescene
             if not io.dir_exists(tree.mve_views):
-                system.run('%s %s %s' % (context.makescene_path, tree.mve_path, tree.smvs))
+                system.run('%s %s %s' % (context.makescene_path, tree.mve_path, tree.mve))
 
             # config
             config = [
-                "-t%s" % self.params.threads,
```

Comment:
We should respect the `--max_concurrency` parameter when invoking dmrecon?

Reply:
Ah yes, this reminds me of the other hitch -- I am not sure dmrecon respects concurrency values. It'd have to be set with OMP: https://github.com/simonfuhrmann/mve/blob/master/apps/dmrecon/dmrecon.cc#L285

Reply:
Interesting, I don't have much experience with OpenMP. Maybe calling `dmrecon` by setting the  `OMP_NUM_THREADS` environment variable could limit them. https://www.ibm.com/support/knowledgecenter/SSGH2K_13.1.3/com.ibm.xlc1313.aix.doc/compiler_ref/ruomprun4.html

Reply:
Yes, I tried that and manually set it to 5 but it used all cores anyway... . But it is c++, there's a good chance I was doing something wrong.

Reply:
This is one I would need help on -- I'm a bit over my head.

Reply:
If the user selects mve as the processing option, and also specifies --max-concurrency, then a warning ("Thread limiting is not available for this option and will run on all threads, are you sure you want to continue? [Y/n]

Reply:
If we can't limit the threads a warning is fine, but I'd caution with the Y/n prompt, could cause nodeodm to get stuck indefinitely.